dags/veda_data_pipeline/groups/processing_tasks.py

```diff
@@ -1,14 +1,17 @@
-from datetime import timedelta
 import json
 import logging
 from copy import deepcopy
 import smart_open
 from airflow.models.variable import Variable
 from airflow.decorators import task
 from veda_data_pipeline.utils.submit_stac import submission_handler
+from veda_data_pipeline.utils.submit_stac_transactions import submit_transactions_handler
 
 group_kwgs = {"group_id": "Process", "tooltip": "Process"}
 
+airflow_vars = Variable.get("aws_dags_variables")
```

Comment:
If we set deserialize_json=True, do you know if we still need to do:
```
var.transactions_endpoint_enabled==true ? "True" : null
```
in main.tf? 

```suggestion
airflow_vars = Variable.get("aws_dags_variables", deserialize_json=True)
```

And maybe we refactor this file a bit so we call Variable.get in tasks rather than in the top level, since the docs seem to imply this is best practice: https://airflow.apache.org/docs/apache-airflow/2.8.4/best-practices.html#airflow-variables

Reply:
I would love a refactor for this (we call the same set of `Variable.get("aws_dags_variables")` lines 16 times in our DAGs), but I think that's a different issue. `.airflowignore` keeps the DAG processor from parsing the `groups` and `utils` folders, so there's no performance loss by having these calls here.

Reply:
I tried using `airflow_vars_json = Variable.get("aws_dags_variables", deserialize_json=True)` and I'm getting a decode error: `requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)`